iot-hub/Samples/module/ModuleSample/Logging/ColorConsoleLogger.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Extensions.Logging;
+using System;
+using System.Globalization;
+using System.Linq;
+
+namespace Microsoft.Azure.Devices.Client.Samples
+{
+    /// <summary>
+    /// The ILogger implementation for writing color log entries to console.
+    /// For additional details, see https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.ilogger?view=dotnet-plat-ext-3.1.
+    /// </summary>
+    public class ColorConsoleLogger : ILogger
+    {
+        private readonly ColorConsoleLoggerConfiguration _config;
+
+        /// <summary>
+        /// Initializes an instance of <see cref="ColorConsoleLogger"/>.
+        /// </summary>
+        /// <param name="name">The category name for each <see cref="ILogger"/> instance. This is usually the class name where the logger is initialized.</param>
```

Comment:
no name param

Reply:
whoops, good catch. This is argument enough to separate out the logger into its own project, I'll do that now.